readme.md

```diff
@@ -28,7 +28,8 @@ npm install -g GoogleChrome/lighthouse
 
 ## Run
 ```sh
-# Start Chrome with a few flags
+# Spawn a subshell in the directory of the globally-installed Lighthouse
```

Comment:
I don't know that the subshell bit is important to communicate
but i totally agree that `npm explore -g xxx -- yyyyy` is rather uncommon and deserves a comment

How about `# Launch chrome by reaching into the lighthouse module to use one of its npm scripts.`